images/win/Windows2016-Azure.json

```diff
@@ -303,6 +303,12 @@
             "scripts":[
                 "{{ template_dir }}/scripts/Installers/Install-7zip.ps1"
             ]
+        },
+        {
+            "type": "powershell",
```

Comment:
Where do we invoke Validate-PostgreSQL.ps1?

Reply:
Thank you. I have added.